containers/Dockerfile.EESSI-client-pilot-centos7

```
@@ -0,0 +1,13 @@
+FROM docker.io/library/centos:7.8.2003
+
+RUN yum install -y http://cvmrepo.web.cern.ch/cvmrepo/yum/cvmfs-release-latest.noarch.rpm \
+ && yum install -y cvmfs cvmfs-config-default cvmfs-fuse3 sudo vim openssh-clients \
+ && curl -OL https://github.com/EESSI/filesystem-layer/releases/download/v0.2.1/cvmfs-config-eessi-0.2.1-1.noarch.rpm \
+ && yum install -y cvmfs-config-eessi-0.2.1-1.noarch.rpm
```

I think we don't need to download this rpm manually (also prevents it from being stored in the root, which is annoying for autocompletion ;-) )?
```suggestion
 && yum install -y https://github.com/EESSI/filesystem-layer/releases/download/v0.2.1/cvmfs-config-eessi-0.2.1-1.noarch.rpm 
```